saleor/graphql/shop/mutations.py

```diff
@@ -34,7 +34,13 @@ class ShopSettingsInput(graphene.InputObjectType):
         description="Default number of max downloads per digital content url"
     )
     default_digital_url_valid_days = graphene.Int(
-        description=("Default number of days which digital content url will be valid")
+        description="Default number of days which digital content url will be valid"
```

Comment:
Let's add a period at the end of each description that we're touching here 😉 👮‍♂️